authentication-ldap/src/test/java/com/synopsys/integration/alert/authentication/ldap/action/LdapManagerTest.java

```diff
@@ -84,16 +84,25 @@ public void testCreateConfiguration() {
         assertEquals(true, expectedLDAPConfigModel.getEnabled());
         assertEquals(DEFAULT_SERVER, expectedLDAPConfigModel.getServerName());
         assertEquals(DEFAULT_MANAGER_DN, expectedLDAPConfigModel.getManagerDn());
-        assertEquals(DEFAULT_MANAGER_PASSWORD, expectedLDAPConfigModel.getManagerPassword().orElse(""));
+        assertTrue(expectedLDAPConfigModel.getManagerPassword().isPresent());
+        assertEquals(DEFAULT_MANAGER_PASSWORD, expectedLDAPConfigModel.getManagerPassword().get());
         assertEquals("", expectedLDAPConfigModel.getAuthenticationType().orElse(""));
-        assertEquals(DEFAULT_REFERRAL, expectedLDAPConfigModel.getReferral().orElse(""));
-        assertEquals(DEFAULT_USER_SEARCH_BASE, expectedLDAPConfigModel.getUserSearchBase().orElse(""));
-        assertEquals(DEFAULT_USER_SEARCH_FILTER, expectedLDAPConfigModel.getUserSearchFilter().orElse(""));
-        assertEquals(DEFAULT_USER_DN_PATTERNS, expectedLDAPConfigModel.getUserDnPatterns().orElse(""));
-        assertEquals(DEFAULT_USER_ATTRIBUTES, expectedLDAPConfigModel.getUserAttributes().orElse(""));
-        assertEquals(DEFAULT_GROUP_SEARCH_BASE, expectedLDAPConfigModel.getGroupSearchBase().orElse(""));
-        assertEquals(DEFAULT_GROUP_SEARCH_FILTER, expectedLDAPConfigModel.getGroupSearchFilter().orElse(""));
-        assertEquals(DEFAULT_GROUP_ROLE_ATTRIBUTE, expectedLDAPConfigModel.getGroupRoleAttribute().orElse(""));
+        assertTrue(expectedLDAPConfigModel.getReferral().isPresent());
+        assertEquals(DEFAULT_REFERRAL, expectedLDAPConfigModel.getReferral().get());
+        assertTrue(expectedLDAPConfigModel.getUserSearchBase().isPresent());
+        assertEquals(DEFAULT_USER_SEARCH_BASE, expectedLDAPConfigModel.getUserSearchBase().get());
+        assertTrue(expectedLDAPConfigModel.getUserSearchFilter().isPresent());
+        assertEquals(DEFAULT_USER_SEARCH_FILTER, expectedLDAPConfigModel.getUserSearchFilter().get());
+        assertTrue(expectedLDAPConfigModel.getUserDnPatterns().isPresent());
+        assertEquals(DEFAULT_USER_DN_PATTERNS, expectedLDAPConfigModel.getUserDnPatterns().get());
+        assertTrue(expectedLDAPConfigModel.getUserAttributes().isPresent());
+        assertEquals(DEFAULT_USER_ATTRIBUTES, expectedLDAPConfigModel.getUserAttributes().get());
+        assertTrue(expectedLDAPConfigModel.getGroupSearchBase().isPresent());
+        assertEquals(DEFAULT_GROUP_SEARCH_BASE, expectedLDAPConfigModel.getGroupSearchBase().get());
+        assertTrue(expectedLDAPConfigModel.getGroupSearchFilter().isPresent());
+        assertEquals(DEFAULT_GROUP_SEARCH_FILTER, expectedLDAPConfigModel.getGroupSearchFilter().get());
+        assertTrue(expectedLDAPConfigModel.getGroupRoleAttribute().isPresent());
+        assertEquals(DEFAULT_GROUP_ROLE_ATTRIBUTE, expectedLDAPConfigModel.getGroupRoleAttribute().get());
     }
```

Comment:
Do you find this suggestion more concise?
```suggestion
    public void testCreateConfiguration() {
        assertFalse(ldapManager.getCurrentConfiguration().isPresent());
        assertFalse(ldapManager.isLdapEnabled());

        LDAPConfigModel ldapConfigModel = createLDAPConfigModel(true, "");
        assertDoesNotThrow(() -> ldapConfigAccessor.createConfiguration(ldapConfigModel));
        LDAPConfigModel expectedLDAPConfigModel = ldapManager.getCurrentConfiguration()
            .orElseThrow(() -> new AssertionFailedError("Expected LDAPConfigModel did not exist"));

        assertTrue(ldapManager.getCurrentConfiguration().isPresent());
        assertEquals(AlertRestConstants.DEFAULT_CONFIGURATION_NAME, expectedLDAPConfigModel.getName());
        assertTrue(ldapManager.isLdapEnabled());
        assertNotEquals(DEFAULT_CONFIG_ID, expectedLDAPConfigModel.getId());
        assertEquals(true, expectedLDAPConfigModel.getEnabled());
        assertEquals(DEFAULT_SERVER, expectedLDAPConfigModel.getServerName());
        assertEquals(DEFAULT_MANAGER_DN, expectedLDAPConfigModel.getManagerDn());
        assertTrue(expectedLDAPConfigModel.getManagerPassword().isPresent());
        assertOptionalField(DEFAULT_MANAGER_PASSWORD, expectedLDAPConfigModel::getManagerPassword);
        assertEquals("", expectedLDAPConfigModel.getAuthenticationType().orElse(""));
        assertOptionalField(DEFAULT_REFERRAL, expectedLDAPConfigModel::getReferral);
        assertOptionalField(DEFAULT_USER_SEARCH_BASE, expectedLDAPConfigModel::getUserSearchBase);
        assertOptionalField(DEFAULT_USER_SEARCH_FILTER, expectedLDAPConfigModel::getUserSearchFilter);
        assertOptionalField(DEFAULT_USER_DN_PATTERNS, expectedLDAPConfigModel::getUserDnPatterns);
        assertOptionalField(DEFAULT_USER_ATTRIBUTES, expectedLDAPConfigModel::getUserAttributes);
        assertOptionalField(DEFAULT_GROUP_SEARCH_BASE, expectedLDAPConfigModel::getGroupSearchBase);
        assertOptionalField(DEFAULT_GROUP_SEARCH_FILTER, expectedLDAPConfigModel::getGroupSearchFilter);
        assertOptionalField(DEFAULT_GROUP_ROLE_ATTRIBUTE, expectedLDAPConfigModel::getGroupRoleAttribute);
    }

    private void assertOptionalField(String expectedValue, Supplier<Optional<String>> fieldSupplier) {
        Optional<String> optional = fieldSupplier.get();
        assertTrue(optional.isPresent());
        assertEquals(expectedValue, optional.get());
    }
```
Encapsulate the check for the optional isPresent and equals into one method to try and make the code more readable.

Reply:
Used changes, but also in additional tests. Check latest commit